lib/private/Template/SCSSCacher.php

```diff
@@ -358,7 +401,7 @@ private function getInjectedVariables(): string {
 			$scss->compile($variables);
 			$this->injectedVariables = $variables;
 		} catch (ParserException $e) {
-			$this->logger->error($e, ['app' => 'core']);
+			$this->logger->error($e->getMessage(), ['app' => 'core']);
```

Comment:
```suggestion
			$this->logger->logException($e, ['app' => 'core']);
```

I missed that one last time :see_no_evil:

Reply:
:heart: